src/cmd/bootstrap.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var path = require('path');
+var nconf = require('nconf');
+var program = require('commander');
+
+var dirname = require('../cli/paths').baseDir;
+var prestart = require('../prestart');
+
+nconf.argv().env({
+	separator: '__',
+});
+
+var env = program.dev ? 'development' : (process.env.NODE_ENV || 'production');
+process.env.NODE_ENV = env;
+global.env = env;
+
+prestart.setupWinston();
+
+// Alternate configuration file support
+var	configFile = path.resolve(dirname, 'config.json');
```

Comment:
Tab instead of space after`var` here

Reply:
Ok, sorry 4785d7499e81907991a13d292a50ae0c7bbace4d